src/unk_0203E880.c

```diff
@@ -701,7 +701,7 @@ BOOL sub_0203F28C (u16 param0)
     return 1;
 }
 
-BOOL sub_0203F2A0 (FieldSystem * fieldSystem, u16 param1)
+BOOL Script_HasBeatenTrainer (FieldSystem * fieldSystem, u16 param1)
 {
     return sub_020507F0(SaveData_Events(fieldSystem->saveData), 1360 + param1);
```

Comment:
Try to document routines like these which are simple if you can name them appropriately. It looks to me like `1360` is just an offset for the save flag, and that `sub_020507F0` checks the value of a particular flag. I'd recommend:

```c
#define TRAINER_DEFEATED_FLAG_OFFSET 1360

BOOL Script_HasBeatenTrainer(FieldSystem *fieldSystem, u16 eventID)
{
    return VarsFlags_IsFlagOn(SaveData_Events(fieldSystem->saveData), TRAINER_DEFEATED_FLAG_OFFSET + eventID);
}
```
Of course, verify behavior when doing this before making such a change.

Reply:
Where would that define go?

Reply:
After `#include` section in the relevant source file for now is fine; we will want to transfer it elsewhere later, but we don't have a central place for important flags yet.